port/zos390/omrvmem.c

```diff
@@ -87,6 +87,10 @@ int get_protectionBits(uintptr_t mode);
 
 #define GET_4K_ALIGNED_ALLOCATION_SIZE(byteAmount) (byteAmount + FOUR_K_MINUS_1 + sizeof(uintptr_t))
 
+#ifndef __MOPL_EXECUTABLE
+#define __MOPL_EXECUTABLE         0x04000000
+#endif
```

Comment:
Missing comment: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif
```suggestion
#endif /* __MOPL_EXECUTABLE */
```